<launch>
    <node name="modelvisualisation" pkg="rviz" type="rviz" output="screen" />
    <node pkg="course" type="zerglings_spawner" name="zerglings_spawner_1" output = "screen">
        <param name="id" type="int" value="1" />
        <param name="x" type="double" value="10"/>
        <param name="y" type="double" value="0" />
        <param name="health" type="int" value="5" />
        <param name="cooldownSpawn" type="int" value="10" />
    </node>
    <node pkg="course" type="zerglings_spawner" name="zerglings_spawner_2" output = "screen">
        <param name="id" type="int" value="2" />
        <param name="x" type="double" value="5"/>
        <param name="y" type="double" value="5" />
        <param name="health" type="int" value="20" />
        <param name="cooldownSpawn" type="int" value="10" />
    </node>
    <node pkg="course" type="zerglings_spawner" name="zerglings_spawner_3" output = "screen">
        <param name="id" type="int" value="3" />
        <param name="x" type="double" value="5"/>
        <param name="y" type="double" value="-5" />
        <param name="health" type="int" value="20" />
        <param name="cooldownSpawn" type="int" value="10" />
    </node>
    <node pkg="course" type="templars_spawner" name="templars_spawner_1" output = "screen">
        <param name="id" type="int" value="1" />
        <param name="x" type="double" value="-5"/>
        <param name="y" type="double" value="0" />
        <param name="countWaves" type="int" value="3" />
        <param name="countUnitInWave" type="int" value="3" />
        <param name="cooldownSpawn" type="int" value="20" />
    </node>
    <node name="kerrigan" pkg="course" type="kerrigan" output="screen" />
</launch>